Derivco.Orniscient/Derivco.Orniscient.Viewer/Controllers/DashboardController.cs

```diff
@@ -23,7 +23,12 @@ public class DashboardController : Controller
             {
                 if (!GrainClient.IsInitialized)
                 {
-                    GrainClient.Initialize(Server.MapPath("~/DevTestClientConfiguration.xml"));
```

Comment:
Grain client is a static. This doesn't look thread safe... Can you initialize the grain client in the startup method for the web app?